docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -0,0 +1,487 @@
+
+[.contract-subindex-inherited]
+.OwnableImpl
+
+* xref:Ownable-owner[`++owner(self)++`]
+* xref:Ownable-renounce_ownership[`++renounce_ownership(self)++`]
+* xref:Ownable-transfer_ownership[`++transfer_ownership(self, new_owner)++`]
+
+--
+
+[.contract-index]
+.Events
+--
+* xref:Ownable-OwnershipTransferred[`++OwnershipTransferred(previous_owner, new_owner)++`]
+--
+
+[#Ownable-Utilities]
+==== Utilities
+
+[.contract-item]
+[[Ownable-assert_only_owner]]
+==== `[.contract-item-name]#++assert_only_owner++#++(self: @ContractState)++` [.item-kind]#internal#
+
+Throws if called by any account other than the owner.
+
+[#Ownable-Functions]
+==== Functions
+
+[.contract-item]
+[[Ownable-initializer]]
+==== `[.contract-item-name]#++initializer++#++(ref self: ContractState, owner: ContractAddress)++` [.item-kind]#internal#
+
+Initializes the contract setting `owner` as the initial owner.
+
+Emits an xref:Ownable-OwnershipTransferred[OwnershipTransferred] event.
+
+[.contract-item]
+[[Ownable-_transfer_ownership]]
+==== `[.contract-item-name]#++_transfer_ownership++#++(ref self: ContractState, new_owner: ContractAddress)++` [.item-kind]#internal#
+
+Transfers ownership of the contract to a new account (`new_owner`).
+Internal function without access restriction.
+
+Emits an xref:Ownable-OwnershipTransferred[OwnershipTransferred] event.
+
+[.contract-item]
+[[Ownable-owner]]
+==== `[.contract-item-name]#++owner++#++(self: @ContractState) → ContractAddress++` [.item-kind]#external#
+
+Returns the address of the current owner.
+
+[.contract-item]
+[[Ownable-renounce_ownership--]]
+==== `[.contract-item-name]#++renounce_ownership++#++(ref self: ContractState)++` [.item-kind]#external#
+
+Leaves the contract without owner. It will not be possible to call
+`assert_only_owner` functions anymore. Can only be called by the current owner.
+
+NOTE: Renouncing ownership will leave the contract without an owner,
+thereby removing any functionality that is only available to the owner.
+
+[.contract-item]
+[[Ownable-transfer_ownership]]
+==== `[.contract-item-name]#++transfer_ownership++#++(ref self: ContractState, new_owner: ContractAddress)++` [.item-kind]#external#
+
+Transfers ownership of the contract to a new account (`new_owner`).
+Can only be called by the current owner.
+
+Emits an xref:Ownable-OwnershipTransferred[OwnershipTransferred] event.
+
+[#Ownable-Events]
+==== Events
+
+[.contract-item]
+[[Ownable-OwnershipTransferred]]
+==== `[.contract-item-name]#++OwnershipTransferred++#++(previous_owner: ContractAddress, new_owner: ContractAddress)++` [.item-kind]#event#
+
+Emitted when the ownership is transferred.
+
+[.contract]
+[[IAccessControl]]
+=== `++IAccessControl++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/05429e4fd34a250ce7a01450190c53275e5c1c0b/src/access/accesscontrol/interface.cairo#L10[{github-icon},role=heading-link]
+
+:grant_role: xref:#IAccessControl-grant_role[grant_role]
```

Comment:
Should we include the IACCESS_CONTROL ID for SRC-5 somewhere in here? I think it'd be a nice benefit for users

Reply:
Another potential spot could be in the `Interface` section in the access page itself